src/ui/mathquill-editor/MathQuillEditorLoadable.js

```diff
@@ -16,7 +16,7 @@ class MathQuillEditorShimmer extends React.PureComponent {
 }
 
 const MathQuillEditorLoadable = Loadable({
-  loader: () => import( './MathQuillEditor'),
+  loader: () => import(/* webpackChunkName: "czi_prosemirror_mathquill" */ './MathQuillEditor'),
```

Comment:
as discussed offline, slight preference for updating the chunk name to include `_editor` so it's easier to map back to the filename